docker/README.md

```diff
@@ -0,0 +1,81 @@
+# Run GreenGuard using Docker
+
+GreenGuard is prepared to be run using [Docker](https://docker.com/).
+
+This are the commands needed to start a Docker container locally that runs a [Jupyter Notebook](
+https://jupyter.org/) already configured to run GreenGuard.
+
+```bash
+docker run -ti -p8888:8888 signals-dev/greenguard:0.2.2.dev0
+```
+
+Further details about the usage of this image can be found [here](
+https://hub.docker.com/repository/docker/signalsdev/greenguard).
+
+## Run GreenGuard on Kubernetes
+
+GreenGuard can also be started using [Kubernetes](https://kubernetes.io/).
+
+Here are the minimum steps to do so:
+
+1. Create a POD yaml file with the these contents:
+
+```yml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: greenguard
+spec:
+  containers:
+  - name: greenguard
+    image: signalsdev/greenguard:0.2.2.dev0
+    ports:
+    - containerPort: 8888
+```
+
+2. Start the POD locally
+
+```bash
+kubectl apply -f pod-file.yml
+```
+
+3. Forward the port 8888
+
+```bash
+kubectl port-forward greenguard 8888
+```
+
+4. Point your browser at http://localhost:8888
+
+On the other hand, if you are planing to run GreenGuard on a distributed service, we provided a
```

Comment:
> **NOTE**: If GreenGuard is run in a production environment we recommend you to use a service and a deployment instead of just a simple POD. You can find a template of this setup [here](docker/greenguard-deployment.yml)